File: Python/6kyu/DuplicateEncoder.py
def duplicate_encode(word):
    # word = list(map(lambda x: x.lower(), word))
    word = word.lower()

    return "".join(["(" if word.count(w) == 1 else ")" for w in word])


print(duplicate_encode("(BfwTLPP!vJ@BYETECVtXRHArbU@VDaGn!"))

File: CSharp/6kyu/DuplicateEncoder.cs
using System.Linq;
using System;

namespace CSharp._6Kyu
{
    public static partial class Kata6
    {
        public static string DuplicateEncode(string word)
        {
            return String.Join("", word.ToLower().Select(w => word.Count(x => x == w) == 1 ? "(" : ")"));
        }
    }
}